sky/provision/kubernetes/instance.py

```diff
@@ -1440,18 +1463,15 @@ def query_instances(
     for pod in pods:
         phase = pod.status.phase
         pod_status = status_map[phase]
+        reason = None
+        if phase in ('Failed', 'Unknown'):
+            reason = _get_pod_termination_reason(pod, cluster_name)
+            logger.debug(f'Pod Status Reason(s): {reason}')
```

Comment:
```suggestion
            logger.debug(f'Pod Status ({phase}) Reason(s): {reason}')
```
just so we can differentiate between unknown and failed